.github/workflows/api.yaml

```diff
@@ -54,11 +54,6 @@ jobs:
       - name: Read Node.js version
         id: node-version
         run: echo "version=$(cat 'galaxy root/client/.node_version')" >> $GITHUB_OUTPUT
-      - uses: actions/setup-node@v5
-        with:
-          node-version: ${{ steps.node-version.outputs.version }}
-          cache: 'yarn'
```

Comment:
We should probably still cache the yarn dependencies with https://github.com/actions/cache/blob/main/examples.md#node---yarn

Reply:
We only use yarn for the client build. Only the selenium and first startup tests should cache yarn